### PR TITLE
Summary AP plot update

### PR DESCRIPTION
**Pull request by varun-nutonomy**
This PR proposes the updates to plots as per our meeting.

For paper, we can probably use `Constr. Veh.` for `construction_vehicle`, but in the dev-kit, we can leave it as `Construction Vehicle`.

With these changes, The AP summary plot would look like this. 
dist_pr_2 0

dist_pr_2 0
 

The old plot looks like this
dist_pr_2 0

### CODE REVIEW

**Comment by holger-nutonomy** [comment]
Looks much better. I suggest to:
- Fix the legend to be NorthEast as our PR curves are usually not there. 
- Always use "Constr. Veh." to have the same appearance everywhere.
- Sort the legend by AP in decreasing order, which makes it easier to figure out the colors for a particular class.

File: python-sdk/nuscenes/eval/detection/constants.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 DETECTION_NAMES = ['car', 'truck', 'bus', 'trailer', 'construction_vehicle', 'pedestrian', 'motorcycle', 'bicycle',
                    'traffic_cone', 'barrier']
+PRETTY_DETECTION_NAMES = {'car': 'Car', 'truck': 'Truck', 'bus': 'Bus', 'trailer': 'Trailer',
+                          'construction_vehicle': 'Construction Vehicle', 'pedestrian': 'Pedestrian',
```
Comment (oscar-nutonomy) [inline python-sdk/nuscenes/eval/detection/constants.py]: Constr. Veh.

Comment (oscar-nutonomy) [reply python-sdk/nuscenes/eval/detection/constants.py]: Add pretty names for attributes also.

**Comment by varun-nutonomy** [comment]
dist_pr_2 0
motorcycle_tp
motorcycle_pr
summary

@oscar-nutonomy @holger-nutonomy @Alex-nutonomy  the final plots look like above. Let me know if anything needs to be changed, thanks.

**Pull request merged**